dockstore-integration-testing/src/test/java/io/dockstore/webservice/ZenodoIT.java

```diff
@@ -427,9 +421,6 @@ void testGenerateDOIFrozenVersion(Hoverfly hoverfly) throws ApiException {
         // Unpublish workflow
         workflowsApi.publish1(workflowId, CommonTestUtilities.createOpenAPIPublishRequest(false));
 
-        // don't die horribly when stubbing something with snapshots, explain the error
-        exception = assertThrows(ApiException.class, () -> workflowsApi.restub(workflowId));
-        assertTrue(exception.getMessage().contains(A_WORKFLOW_MUST_HAVE_NO_SNAPSHOT_TO_RESTUB));
 
         // Publish workflow
         workflowsApi.publish1(workflowId, CommonTestUtilities.createOpenAPIPublishRequest(true));
```

Comment:
Can remove the `Unpublish workflow` and `Publish workflow` lines since they were only there for restubbing